x-pack/legacy/plugins/siem/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -223,5 +232,24 @@ export const getNetworkTopNFlowColumns = (
   },
 ];
 
-const getOppositeField = (flowTarget: FlowTargetNew): FlowTargetNew =>
-  flowTarget === FlowTargetNew.source ? FlowTargetNew.destination : FlowTargetNew.source;
+export const getNFlowColumnsCurated = (
+  indexPattern: StaticIndexPattern,
+  flowTarget: FlowTargetSourceDest,
+  type: networkModel.NetworkType,
+  tableId: string
+): NetworkTopNFlowColumns => {
```

Comment:
Should this return type be the union of the two column types?

``` ts
NetworkTopNFlowColumns | NetworkTopNFlowColumnsIpDetails
```

Reply:
good catch